client/tests/__mocks__/GraphQLMocks/TestPlanReportStatusDialogMock.js

```diff
@@ -827,7 +827,12 @@ export default (
         me: {
           id: '1',
           username: 'foo-bar',
-          roles: ['ADMIN', 'TESTER']
+          roles: ['ADMIN', 'TESTER'],
+          company: {
+            id: '1',
+            name: 'Company',
+            ats: []
+          }
```

Comment:
in the mock, would these need a company since the roles assigned here doesn't include 'VENDOR'?

Reply:
Great point! Addressed.